examples/packages/WORKSPACE

```diff
@@ -23,6 +23,12 @@ npm_install(
     package_lock_json = "//:package-lock.json",
 )
 
+npm_install(
```

Comment:
add comments explaining the purpose of these duplicates

Reply:
still think a comment is needed or is the run name as `npm_install_duplicate_for_determinism_testing` sufficient?

Reply:
good enough w/o comment